terraform/production/main.tf

```diff
@@ -96,24 +96,22 @@ provider "github" {
 module "production_cluster" {
   source = "../modules/spack"
 
-  deployment_name = "production"
+  deployment_name = "prod"
 
   availability_zones = ["us-east-1a", "us-east-1b", "us-east-1c", "us-east-1d"]
 
-  vpc_cidr = "10.0.0.0/16"
+  vpc_cidr = "192.168.0.0/16"
   public_subnets = [
-    "10.0.0.0/19",
-    "10.0.32.0/19",
+    "192.168.0.0/19",
+    "192.168.32.0/19",
+    "192.168.64.0/19",
+    "192.168.96.0/19",
   ]
   private_subnets = [
-    "10.0.64.0/19",
-    "10.0.96.0/19",
-    "10.0.128.0/19",
-    "10.0.160.0/19",
-  ]
-  database_subnets = [
-    "10.0.192.0/19",
-    "10.0.224.0/19",
+    "192.168.128.0/19",
+    "192.168.160.0/19",
+    "192.168.192.0/19",
+    "192.168.224.0/19",
   ]
```

Comment:
One thing I forgot to ask, why the change in these values?

Reply:
I copied the old VPC config (from before the NAT gateway's were messed up) from [here](https://github.com/spack/spack-infrastructure/blob/91395c7d20a9bb50e48ec27133a3cc428a6df879/scripts/eks-vpc.yaml) as closely as possible. But yeah it wasn't actually necessary to change those values.